src/web/client/test/integration/commonUtil.test.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import { expect } from "chai";
+import { schemaEntityName } from "../../schema/constants";
+import {
+    convertfromBase64ToString,
+    convertStringtoBase64,
+    GetFileNameWithExtension,
+} from "../../utilities/commonUtil";
+
+describe("commonUtil", async () => {
+    it("encodeBase64_And_DecodeBase64_string", () => {
```

Comment:
proper naming missing and don't mix two methods in a single test case.
Write different test cases

Reply:
ok